<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeWindow</class>
 <widget class="QWidget" name="HomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>646</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HomeWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: rgb(62, 62, 62);
	color: rgb(255, 255, 255);
}

QToolTip   {
	border: 1px solid black;
	background-color: rgb(255, 213, 61);
	color: rgb(0, 0, 0);
}
QPushButton{
	background:transparent;
}
QPushButton:hover{
	background-color:rgb(75, 75, 75);
}
QPushButton:pressed{
	background-color:rgb(75, 75, 75);
}
QFrame{
	border-color: #828790;
}
QTableWidget{
	border-color: #828790;
}
QTreeWidget{
	border-color: #828790;
}
QLineEdit{
	border-color: #828790;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QScrollArea" name="HeaderBlock">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>75</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>75</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QScrollArea::frame { border: 0; }</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="HeaderBlockScrollArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1006</width>
        <height>75</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>75</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>75</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>15</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="HeaderBlockBtnAddBook">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/white/book_add.png</normaloff>../icons/white/book_add.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="HeaderBlockBtnDelBook">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/white/book_del.png</normaloff>../icons/white/book_del.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="HeaderBlockScrollFiller">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>427</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="HeaderBlockBtnSettings">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/white/settings.png</normaloff>../icons/white/settings.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="MainBlock">
     <item>
      <layout class="QVBoxLayout" name="SortingBlock">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <widget class="QTreeWidget" name="SortingBlockTree">
         <property name="minimumSize">
          <size>
           <width>210</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>210</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="rootIsDecorated">
          <bool>true</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="itemsExpandable">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>false</bool>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Groupes</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>Tout</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Series</string>
          </property>
          <item>
           <property name="text">
            <string>Harry Potter</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Gundam</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>Auteurs</string>
          </property>
          <item>
           <property name="text">
            <string>truc</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>machin</string>
           </property>
          </item>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="SortingBlockSearchBlock">
         <item>
          <widget class="QLabel" name="SortingBlockSearchLabel">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="SortingBlockSearchValue">
           <property name="maximumSize">
            <size>
             <width>170</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: #828790 1px solid;</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="InfoBlock_2">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
	color: #828790;
	border-color: #828790;
	border-top-width: 0px;
}
QLabel{
	color: rgb(255, 255, 255);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QVBoxLayout" name="InfoBlock" stretch="0,0,0,0,0,0,0">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item>
           <widget class="QPushButton" name="InfoBlockCover">
            <property name="minimumSize">
             <size>
              <width>190</width>
              <height>160</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>190</width>
              <height>160</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background: transparent !important;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>160</width>
              <height>160</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="InfoBlockTitle">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <item>
             <widget class="QLabel" name="InfoBlockTitleLabel">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="InfoBlockTitleValue">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="InfoBlockSerie">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <item>
             <widget class="QLabel" name="InfoBlockSerieLabel">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="InfoBlockSerieValue">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="InfoBlockAuthors">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <item>
             <widget class="QLabel" name="InfoBlockAuthorsLabel">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="InfoBlockAuthorsValue">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="InfoBlockFileFormats">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <item>
             <widget class="QLabel" name="InfoBlockFileFormatsLabel">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="InfoBlockFileFormatsValue">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>&lt;a href=&quot;test&quot;&gt;test&lt;/a&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="InfoBlockSize">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <item>
             <widget class="QLabel" name="InfoBlockSizeLabel">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="InfoBlockSizeValue">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="InfoBlockSynopsis">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="CentralBlock">
       <item>
        <widget class="QTableWidget" name="CentralBlockTable">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">::section {
spacing: 10px;
background-color: #333333;
color: white;
border: 1px solid black;
margin: 0;
text-align: right;
font-family: arial;
font-size: 12px; }
QTableView::item { padding: 0px }

QTableView::item {cursor:pointer;}
QTableView::item:hover {background: skyblue;}
QTableView::item:selected {background: lightgray; color: black;}</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideRight</enum>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>0</number>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>1</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>2</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Title</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Author</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Serie</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tags</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Modified</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>aa</string>
          </property>
          <property name="textAlignment">
           <set>AlignLeading|AlignVCenter</set>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>bb</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>ccc</string>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string>ddd</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
